po/ja.po

```diff
@@ -14374,6 +14389,24 @@ msgid ""
 "* `.await` can only be used inside an `async` function (or block; these are\n"
 "  introduced later). "
 msgstr ""
+"* これは構文を示すための単純化された例であることに注意してください。長く実行されうる操作や本物の並"
+"行処理はここには含まれません。\n"
+"* 非同期の呼び出しの帰り値の型は？\n"
+"  * 型を知るために`main`で`let future: () = async_main(10);`を使ってみてください。\n"
+"\n"
+"* 「async」キーワードは糖衣構文です。コンパイラは帰り値をfutureに置き換えます。\n"
+"\n"
+"* コンパイラに対して、返却されたfutureの値をその後どう扱うべきかという、追加の指示を含めない限り、"
+"`main` asyncを作ることはできません。\n"
+"\n"
+"* エグゼキュータが非同期のコードを実行するために必要です。`block_on`は、与えられたfutureが最後まで"
+"実行されるまで、現在のスレッドをブロックします。\n"
+"\n"
+"* `.await`は非同期的に他の操作の完了を待ちます。`block_on`とは異なり、`.await`は現在のスレッドをブ"
+"ロックしません。\n"
+"\n"
+"* `.await`は`async`の関数の中でのみ利用できます。（もしくはブロックについても同様ですが、これは後ほ"
```

Comment:
「もしくはブロックについても同様ですが」: it's not so clear what this "ブロック" means here. Especially, since we explained `block_on` above, which is a different type of "block", it may be more confusing.

So, I'd make it more verbose to make it less confusing:
「`.await`は`async`関数(または`async` ブロック)の中でのみ利用できます。(`async`関数・ブロックについては後ほど紹介します。）」

async function and async block will be explained in https://google.github.io/comprehensive-rust/async/tasks.html